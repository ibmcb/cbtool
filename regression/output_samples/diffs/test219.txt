|arrival                                 |1374526226                           
|cloud_hostname                          |3814B750-188E-51A0-A67E-9E3B7C0E32B9.simcloud.com
|cloud_ip                                |254.56.139.236                       
|cloud_mac                               |00:16:3e:23:dd:4e                    
|cloud_vm_name                           |cb-msilva-vm_1-tinyvm                
|cloud_vm_uuid                           |3814B750-188E-51A0-A67E-9E3B7C0E32B9 
|command_originated                      |1374526224                           
|host                                    |02A87729-F726-5F81-8D0D-7F3B61CF358F 
|host_cloud_ip                           |24.169.15.34                         
|host_name                               |simhosta0                            
|mgt_001_provisioning_request_originated |1374526224                           
|prov_cloud_ip                           |254.56.139.236                       
|uuid                                    |64DD0F5F-27CF-50C8-AE2A-0930A713B586 
|vmc                                     |CA6252C4-6803-5C5C-BAA8-5BC509EE3CE0 
|vmc_cloud_ip                            |8.188.4.54                           
