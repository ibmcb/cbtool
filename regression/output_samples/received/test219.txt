The VM object vm_1, attached to this experiment, has the following attributes (Cloud TESTCLOUD) :
|attribute (VM object key)               |value                                
|access                                  |                                     
|ai                                      |none                                 
|ai_arrived                              |0                                    
|ai_arriving                             |0                                    
|ai_departed                             |0                                    
|ai_failed                               |0                                    
|ai_name                                 |none                                 
|ai_reservations                         |0                                    
|aidrs                                   |none                                 
|aidrs_name                              |none                                 
|arrival                                 |1374526226                           
|attempts                                |3                                    
|base_dir                                |/home/msilva/cloudbench/lib/auxiliary//../..
|capture_supported                       |True                                 
|check_boot_complete                     |wait_for_0                           
|check_boot_started                      |poll_cloud                           
|class                                   |standard                             
|client_pref_pool                        |lg                                   
|cloud_hostname                          |3814B750-188E-51A0-A67E-9E3B7C0E32B9.simcloud.com
|cloud_ip                                |254.56.139.236                       
|cloud_mac                               |00:16:3e:23:dd:4e                    
|cloud_name                              |TESTCLOUD                            
|cloud_vm_name                           |cb-msilva-vm_1-tinyvm                
|cloud_vm_uuid                           |3814B750-188E-51A0-A67E-9E3B7C0E32B9 
|command                                 |vmattach TESTCLOUD tinyvm auto empty default none empty=empty
|command_originated                      |1374526224                           
|counter                                 |89                                   
|cpu_lower                               |500                                  
|cpu_upper                               |1000                                 
|credentials                             |                                     
|daemon_dir                              |/home/klabuser                       
|debug_remote_commands                   |False                                
|detach_parallelism                      |20                                   
|discover_hosts                          |True                                 
|driver_pref_pool                        |lg                                   
|eclipsed                                |False                                
|exclude_list                            |/home/msilva/cloudbench/lib/auxiliary//../../exclude_list.txt
|execute_json_filename_prefix            |cb                                   
|execute_script_name                     |execute_on_staging.sh                
|experiment_id                           |REGRESSION_B                         
|extra_info                              |                                     
|host                                    |02A87729-F726-5F81-8D0D-7F3B61CF358F 
|host_cloud_ip                           |24.169.15.34                         
|host_name                               |simhosta0                            
|hosts_per_vmc                           |5                                    
|identity                                |/home/msilva/cloudbench/lib/auxiliary//../../credentials/bogus_id_rsa
|imageid1                                |cloudbench_tinyvm                    
|imageids                                |1                                    
|jars_dir                                |/home/klabuser/cloudbench/jar        
|key_name                                |                                     
|last_known_state                        |running with ip assigned             
|login                                   |klabuser                             
|memory_lower                            |50                                   
|memory_upper                            |100                                  
|meta_tags                               |empty                                
|mgt_001_provisioning_request_originated |1374526224                           
|mgt_002_provisioning_request_sent       |0                                    
|mgt_003_provisioning_request_completed  |1                                    
|mgt_004_network_acessible               |1                                    
|mgt_005_file_transfer                   |0                                    
|mgt_006_application_start               |0                                    
|mode                                    |controllable                         
|model                                   |sim                                  
|name                                    |vm_1                                 
|netid                                   |vlan123                              
|notification                            |False                                
|pattern                                 |none                                 
|prov_cloud_ip                           |254.56.139.236                       
|qemu_debug                              |False                                
|randomize_image_name                    |False                                
|remote_dir_name                         |cloudbench                           
|resize_supported                        |False                                
|role                                    |tinyvm                               
|run_generic_scripts                     |False                                
|runstate_parallelism                    |5                                    
|runstate_supported                      |True                                 
|security_groups                         |                                     
|size                                    |pico32                               
|ssh_key_name                            |bogus_id_rsa                         
|staging                                 |none                                 
|state                                   |attached                             
|svm_stub                                |False                                
|svm_stub_vmc                            |none                                 
|temp_attr_list                          |empty=empty                          
|timeout                                 |240                                  
|tracking                                |none                                 
|transfer_files                          |False                                
|type                                    |none                                 
|update_attempts                         |720                                  
|update_frequency                        |1                                    
|username                                |msilva                               
|uuid                                    |64DD0F5F-27CF-50C8-AE2A-0930A713B586 
|vcpus                                   |1                                    
|vm_arrived                              |0                                    
|vm_arriving                             |0                                    
|vm_departed                             |0                                    
|vm_failed                               |0                                    
|vm_reservations                         |0                                    
|vmc                                     |CA6252C4-6803-5C5C-BAA8-5BC509EE3CE0 
|vmc_access                              |                                     
|vmc_arrived                             |16                                   
|vmc_cloud_ip                            |8.188.4.54                           
|vmc_departed                            |12                                   
|vmc_failed                              |0                                    
|vmc_name                                |simzone_a                            
|vmc_pool                                |SUT                                  
|vmemory                                 |192                                  
|vnics                                   |1                                    
|vstorage                                |2048                                 

