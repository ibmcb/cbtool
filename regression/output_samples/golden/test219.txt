The VM object vm_1, attached to this experiment, has the following attributes (Cloud TESTCLOUD) :
|attribute (VM object key)               |value                                
|access                                  |                                     
|ai                                      |none                                 
|ai_arrived                              |0                                    
|ai_arriving                             |0                                    
|ai_departed                             |0                                    
|ai_failed                               |0                                    
|ai_name                                 |none                                 
|ai_reservations                         |0                                    
|aidrs                                   |none                                 
|aidrs_name                              |none                                 
|arrival                                 |1372701468                           
|attempts                                |3                                    
|base_dir                                |/home/msilva/cloudbench/lib/auxiliary//../..
|capture_supported                       |True                                 
|check_boot_complete                     |wait_for_0                           
|check_boot_started                      |poll_cloud                           
|class                                   |standard                             
|client_pref_pool                        |lg                                   
|cloud_hostname                          |82920618-E014-51AB-AC29-5BE1C078BB9B.simcloud.com
|cloud_ip                                |122.45.84.64                         
|cloud_mac                               |00:16:3e:1e:45:71                    
|cloud_name                              |TESTCLOUD                            
|cloud_uuid                              |82920618-E014-51AB-AC29-5BE1C078BB9B 
|cloud_vm_name                           |cb-msilva-vm_1-tinyvm                
|command                                 |vmattach TESTCLOUD tinyvm auto empty default none empty=empty
|command_originated                      |1372701466                           
|counter                                 |89                                   
|cpu_lower                               |500                                  
|cpu_upper                               |1000                                 
|credentials                             |                                     
|daemon_dir                              |/home/klabuser                       
|debug_remote_commands                   |False                                
|detach_parallelism                      |20                                   
|discover_hosts                          |True                                 
|driver_pref_pool                        |lg                                   
|eclipsed                                |False                                
|exclude_list                            |/home/msilva/cloudbench/lib/auxiliary//../../exclude_list.txt
|execute_json_filename_prefix            |cb                                   
|execute_script_name                     |execute_on_staging.sh                
|experiment_id                           |REGRESSION_B                         
|extra_info                              |                                     
|host                                    |0AC8DBE3-E7A7-56E6-8731-27D204729587 
|host_cloud_ip                           |250.167.11.199                       
|host_name                               |simhosta2                            
|hosts_per_vmc                           |5                                    
|identity                                |/home/msilva/cloudbench/lib/auxiliary//../../credentials/bogus_id_rsa
|imageid1                                |cloudbench_tinyvm                    
|imageids                                |1                                    
|jars_dir                                |/home/klabuser/cloudbench/jar        
|key_name                                |                                     
|last_known_state                        |running with ip assigned             
|login                                   |klabuser                             
|memory_lower                            |50                                   
|memory_upper                            |100                                  
|meta_tags                               |empty                                
|mgt_001_provisioning_request_originated |1372701466                           
|mgt_002_provisioning_request_sent       |0                                    
|mgt_003_provisioning_request_completed  |1                                    
|mgt_004_network_acessible               |1                                    
|mgt_005_file_transfer                   |0                                    
|mgt_006_application_start               |0                                    
|mode                                    |controllable                         
|model                                   |sim                                  
|name                                    |vm_1                                 
|netid                                   |vlan123                              
|notification                            |False                                
|pattern                                 |none                                 
|prov_cloud_ip                           |122.45.84.64                         
|qemu_debug                              |False                                
|randomize_image_name                    |False                                
|remote_dir_name                         |cloudbench                           
|resize_supported                        |False                                
|role                                    |tinyvm                               
|run_generic_scripts                     |False                                
|runstate_parallelism                    |5                                    
|runstate_supported                      |True                                 
|security_groups                         |                                     
|size                                    |pico32                               
|ssh_key_name                            |bogus_id_rsa                         
|staging                                 |none                                 
|state                                   |attached                             
|svm_stub                                |False                                
|svm_stub_vmc                            |none                                 
|temp_attr_list                          |empty=empty                          
|timeout                                 |240                                  
|tracking                                |none                                 
|transfer_files                          |False                                
|type                                    |none                                 
|update_attempts                         |720                                  
|update_frequency                        |1                                    
|username                                |msilva                               
|uuid                                    |E68D0F85-F4DE-52F9-909E-D7A869F3B6C6 
|vcpus                                   |1                                    
|vm_arrived                              |0                                    
|vm_arriving                             |0                                    
|vm_departed                             |0                                    
|vm_failed                               |0                                    
|vm_reservations                         |0                                    
|vmc                                     |97FF48D4-05C4-54D7-B1D8-828EED2A2B20 
|vmc_access                              |                                     
|vmc_arrived                             |16                                   
|vmc_cloud_ip                            |77.193.103.115                       
|vmc_departed                            |12                                   
|vmc_failed                              |0                                    
|vmc_name                                |simzone_a                            
|vmc_pool                                |SUT                                  
|vmemory                                 |192                                  
|vnics                                   |1                                    
|vstorage                                |2048                                 

